CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to this repo
+
+## Setup
+
+### Personal access token (PAT)
+
+Follow [these](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) instructions to configurate a personal access token.
+
+Only permissions needed:
+- public_repo
+
+Save the token to a secure location. Click the "Configure SSO" button and "Authorize" if applicable.
+
+### Config
+
+```shell
+cp config.example.json config.json
+
+# Add your GitHub username and token
+$EDITOR config.json
+
+# Export the JSON credentials into an environment variable
+export CONFIG=$(<config.json)
```

Comment:
I'm unfamiliar with this syntax, so just checking that the lonely `<` is meant to be there?